bootstrap.go

```diff
@@ -30,9 +30,14 @@ func bootstrap(c *cli.Context) {
 		log.Fatalf("[bootstrap] --consulServers is required. See '%s bootstrap --help'.", c.App.Name)
 	}
 
+	if len(c.String(flConsulLogLocation.Name)) == 0 {
```

Comment:
I believe it shall  not be mandatory parameter. If you don't want to collect logs then we do not care.